### PR TITLE
docs: add langchain evaluation notebook

### PR DESCRIPTION
**Pull request by jordanrfrazier**
Adds a basic langchain evaluation notebook using langchain evaluators and langsmith.

### CODE REVIEW

File: langchain_evaluation.ipynb [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "You will need a vector-enabled Astra database and an OpenAI Account.\n",
+    "\n",
+    "* Create an [Astra vector database](https://docs.datastax.com/en/astra-serverless/docs/getting-started/create-db-choices.html).\n",
+    "* Create an [OpenAI account](https://openai.com/)\n",
+    "* Within your database, create an [Astra DB keyspace](https://docs.datastax.com/en/astra-serverless/docs/manage/db/manage-keyspaces.html#_adding_a_new_keyspace)\n",
```
Comment (cdbartholomew) [inline langchain_evaluation.ipynb:31]: With the new vector experience, a keyspace is automatically created for you, so I am not sure we need to include this.

Comment (jordanrfrazier) [reply langchain_evaluation.ipynb:31]: Okay cool -- I can omit this and mention that a `default_keyspace` is created if it's needed later

File: langchain_evaluation.ipynb [hunk 2]
```diff
@@ -0,0 +1,653 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "`ragstack-ai` includes all the packages you need to build a RAG pipeline. \n",
+    "\n",
+    "`datasets` is used to import a sample dataset\n",
+    "\n",
+    "`langchain[openai]` is used to bring in LangSmith"
```
Comment (epinzur) [inline langchain_evaluation.ipynb:49]: surprised you need this... isn't it included with `ragstack-ai`?

**Review by mendonk — APPROVED** [review]
I don't have a Langsmith key but this looks like it should work

File: langchain_evaluation.ipynb (hunk 1 above)
Comment (epinzur) [reply langchain_evaluation.ipynb:31]: I think you can drop talking about keyspaces alltogether.  they don't exist as a configurable thing in the new experience.

File: langchain_evaluation.ipynb [hunk 3]
```diff
@@ -0,0 +1,653 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "`ragstack-ai` includes all the packages you need to build a RAG pipeline. \n",
+    "\n",
+    "`datasets` is used to import a sample dataset\n",
```
Comment (epinzur) [inline langchain_evaluation.ipynb:47]: doesn't seem like you are using this

File: langchain_evaluation.ipynb (hunk 2 above)
Comment (jordanrfrazier) [reply langchain_evaluation.ipynb:49]: You're correct, looks like we do.

File: langchain_evaluation.ipynb (hunk 1 above)
Comment (jordanrfrazier) [reply langchain_evaluation.ipynb:31]: Oh yeah good catch. Removed `keyspace` entirely.

**Pull request merged**